.github/workflows/wpt-test.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: WebPageTest Test Cases
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: npm install webpagetest && npm install jest
+
+      - name: Run WebPageTest
+        run: |
+          FILES=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} \
+            ${{ github.event.pull_request.head.sha }} | grep -E "^dist/.*\.js$")
+          for FILE in $FILES
+          do
+            TEST_NAME=${FILE:5:-3}
+            echo "Running test: $TEST_NAME"
+
+            if [ -f "tests/$TEST_NAME.test.js" ]
+            then
+              npm run test -- $TEST_NAME
+            else
+              echo "There are no test cases for $TEST_NAME"
+              exit 1
+            fi
+          done
+        env:
+          WPT_API_KEY: ${{ secrets.WPT_API_KEY }}
```

Comment:
@rviscomi could you add a WPT secret as expected here

Reply:
And if you could share one to speed up local dev - would be great!

Reply:
Set up the `WPT_API_KEY` repo secret and DM'd you in the HTTP Archive slack with a private key you can use for testing.